### DISCUSSION TITLE
Seshat Databank

### DISCUSSION THREAD

**Discussion by rybesh** [periodo/periodo-data#94] [New period authorities]
The [Seshat Databank](http://seshatdatabank.info/databrowser/) appears to have a number of periodizations suitable for inclusion in PeriodO. See for example the periodizations of the [Cambodian Basin](http://seshatdatabank.info/databrowser/cambodian-basin.html) or the [Finger Lakes](http://seshatdatabank.info/databrowser/finger-lakes.html). 

What do you think @atomrab?

**Comment by atomrab** [comment]
I'm reluctant to incorporate these values without more discussion with SESHAT -- we started a conversation several years ago, but for whatever reason it ran aground. I think that what looks like periodization here is something a bit more idiosyncratic, where they're focused on the dates of polities as defined in a way that allows them to quantify various developmental variables. Here's the statement on temporal boundaries: 

> The next three coding positions define the temporal bounds of the polity. These codes take into account that such temporal bounds may be fuzzy and allow us to capture this 'fuzziness.' For example, some polities such as the Medieval German Empire or China under the Zhou Dynasty began as reasonably coherent states, but with time gradually lost cohesion, the degree to which the center exercized control over regional subpolities. Because this process was gradual, there was no sharp temporal boundary. The 'Degree of centralization' variable allows to capture these transitions (by coding time periods when the polity transitions, for example, from a 'confederated state' to 'loose' and finally to 'nominal' degree of centralization. Similarly, polities may have a fuzzy starting date, if they originate as subpolities under a disintegrating overarching polity. These transitions are captured by the variable 'Supra-polity relations'.

We could, however, look at the sources they're using (usually reasonably recent synthetic histories, at least at a glance) to fill in our gaps for particular areas, if that seems useful. Or we could work with them to figure out if what we're doing makes sense with what they're doing, and then arrange a way to import their material where our understandings seem to coincide. I'd tend to prefer that; I can reestablish contact with Daniel Hoyer, with whom we began the conversation, if this seems like the way to go.
